### ISSUE TITLE
Option to specify `theme-color`

### ISSUE THREAD

**Issue by donovanglover** [welpo/tabi#137]
# Feature Request

## Summary
Chromium on Android and other browsers change the UI appearance based on `theme-color`.

## Motivation
`theme-color` results in a more consistent brand that improves the overall impression of the website.

## Detailed Description
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color

**Comment by donovanglover** [comment]
Not sure if anything changed but last time I did this [CSS variables weren't supported](https://css-tricks.com/meta-theme-color-and-trickery/#aa-custom-properties). Might need to use a config value instead.

**Comment by welpo** [comment]
Very cool! I was not aware of this element.

> Not sure if anything changed but last time I did this [CSS variables weren't supported](https://css-tricks.com/meta-theme-color-and-trickery/#aa-custom-properties).

Indeed, I just tested it and CSS variables aren't supported.

It would be possible to use JavaScript to update the value, but I'd rather not add JS for such a minor feature.

In fact, even then, that would not be very helpful with the default values. If your colour si too bright, (at least) Safari will ignore the value when the OS is set to dark. And Safari considers the primary-color too bright on dark mode (not just for the default teal theme, but for all the skins I tested too).
